package.json

```diff
@@ -12,23 +12,16 @@
     "lint": "gulp lint"
```

Comment:
In the feature, you could create a script for dependencies verify, using something like: `npm-check` :)